File: Leet Code/JavaScript/problem_58.js
const lengthOfLastWord = function(s) {
    let myStr = s;
    myStr = myStr.trim()
    let lastSpaceIndex = myStr.lastIndexOf(' ')
    if (lastSpaceIndex === -1) {
        return myStr.length;
    }
    return myStr.length - lastSpaceIndex - 1;
};

let inputStr = "h";
inputStr.length;
const result = lengthOfLastWord(inputStr);
console.log("the length of the last word in string is ", result);

File: Leet Code/Python/problem_58.py
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        myStr = s.strip()
        lastWord = myStr.rfind(' ')

        if (lastWord < 0) :
           return len(myStr)
        
        return len(myStr) - lastWord - 1


s = Solution()
inputStr = input("Enter anything: ")
lastWordCount = s.lengthOfLastWord(inputStr)
print("The length of last word in string entered is ", lastWordCount)